apps/opencs/model/world/refidadapterimp.cpp

```diff
@@ -431,7 +438,15 @@ void CSMWorld::MiscRefIdAdapter::setData (const RefIdColumn *column, RefIdData&
         InventoryRefIdAdapter<ESM::Miscellaneous>::setData (column, data, index, value);
 }
 
-CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns) : ActorColumns (actorColumns) {}
+CSMWorld::NpcColumns::NpcColumns (const ActorColumns& actorColumns)
+: ActorColumns (actorColumns),
+  mFlags(std::map<const RefIdColumn *, unsigned int>()),
```

Comment:
The compiler should catch such cases, if anyone changes the type of the variable or removes it.

Anyway, I removed it in a new commit: 44e01d0eaaad9
Include it, if you want to get rid of that line. Leave it out, if you don't.